spec/models/course_module_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CourseModule, type: :model do
+  it "can be created" do
+    expect {
+      CourseModule.create(
+        title: "Test Course module",
+        content: "No content",
+        course_year: FactoryBot.create(:course_year),
+      )
+    }.to change { CourseModule.count }.by(1)
+  end
+
+  describe "associations" do
+    it { is_expected.to belong_to(:course_year) }
```

Comment:
Same here, I think we can add similar checks for the other associations in the class